packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CheckoutConfirm/template.success.tsx

```diff
@@ -264,7 +264,12 @@ const Success: React.FC<InjectedFormProps<{ form: string }, Props> & Props> = (p
               </IconWrapper>
             </RowIcon>
             <RowText data-e2e='sbExchangeRate'>
-              {displayFiat(props.order, props.quote.rate.toString())}
+              {displayFiat(
+                props.order,
+                props.isFlexiblePricingModel
+                  ? (1 / (parseInt(props.quote.rate.toString()) * (1 / 10000000000))).toString()
```

Comment:
maybe to move this to function , also 1/10000000000 you can write just like `0.0000000001` move to constant and add comment what it represents